src/devices/bus/coco/cococart.cpp

```diff
@@ -48,6 +48,18 @@
 #define LOG_LINE                0
 
 
+/***************************************************************************
+	CONSTANTS
+***************************************************************************/
+
+enum
+{
+	TIMER_CART,
+	TIMER_NMI,
+	TIMER_HALT
+};
```

Comment:
Then make it a final class to prevent it or something.  The whole timer IDs thing is dangerous at the best of times.